buildscripts/builder.sh

```diff
@@ -0,0 +1,179 @@
+OPENSSL_VERSION=1_0_2s
+DIRECTORY32BIT=SoftwareDownloads32bit
+DIRECTORY64BIT=SoftwareDownloads64bit
+
+if [ ${MACHINE_TYPE} == 'x86_64' ]; then
+	if [ ! -d "$DIRECTORY64BIT" ]; then
+		mkdir SoftwareDownloads64bit
+        cd SoftwareDownloads64bit
+    else
+    	echo "Directory already exists"
+    	cd SoftwareDownloads64bit
+	fi
+else
+	if [ ! -d "$DIRECTORY32BIT" ]; then
+		mkdir SoftwareDownloads32bit
+        cd SoftwareDownloads32bit
+    else
+    	echo "Directory 32 bit alrready exists"
+    	cd SoftwareDownloads32bit
+	fi
+fi
+#Functions
+function install_wine {
+
+	
+	wget -nc https://dl.winehq.org/wine-builds/Release.key  --no-check-certificate
+	sudo apt-key add Release.key
+	sudo apt-add-repository 'https://dl.winehq.org/wine-builds/ubuntu/'
+	sudo apt-get -y update
+	sudo apt-get install wine1.8
+	sudo apt-get install wine1.8 winetricks
+	if [ ${MACHINE_TYPE} == 'x86_64' ]; then
+		sudo apt-get install wine64-development
+		env WINEPREFIX=$HOME/.wine64 WINEARCH=win64 winecfg
+		WINE="env WINEPREFIX=$HOME/.wine64 wine"
+		export WINEPREFIX
+		
+	else
+		sudo apt-get install wine32-development
+		env WINEPREFIX=$HOME/.wine32 WINEARCH=win32 winecfg
+		WINE="env WINEPREFIX=$HOME/.wine32 wine"
+		export WINEPREFIX
+		
+	fi
+}
+
+function install_python(){
+	echo "Download Python2.7"
+	
+	if [ ${MACHINE_TYPE} == 'x86_64' ]; then
+		# For 64 bit machine
+		wget -nc wget http://www.python.org/ftp/python/${PYTHON_VERSION}/python-${PYTHON_VERSION}.amd64.msi --no-check-certificate
+		echo "Install Python2.7 for 64 bit"
+  		$WINE msiexec -i python-${PYTHON_VERSION}.amd64.msi /qn
+		
+		wget -nc https://download.microsoft.com/download/d/2/4/d242c3fb-da5a-4542-ad66-f9661d0a8d19/vcredist_x64.exe --no-check-certificate
+		$WINE vcredist_x64.exe		
+		echo "Installed vcredist for 64 bit"
+		$WINE pip install --upgrade pip
+
+	else
+		# For 32 bit machine
+		cd SoftwareDownloads32bit
+		wget -nc https://www.python.org/ftp/python/${PYTHON_VERSION}/python-${PYTHON_VERSION}.msi --no-check-certificate
+		echo "Install Python2.7 for 32 bit"
+  		$WINE msiexec -i python-${PYTHON_VERSION}.msi /qn
+
+  		echo "Installing vc_redist for 32 bit " 		
+		wget -nc https://download.microsoft.com/download/1/1/1/1116b75a-9ec3-481a-a3c8-1777b5381140/vcredist_x86.exe --no-check-certificate
+  		$WINE vcredist_x86.exe
+  		
+  		#insatlled msvcr120.dll for 32 bit system 
+  		wget -nc http://www.dll-found.com/zip/m/msvcr120.dll.zip --no-check-certificate
+		unzip msvcr120.dll.zip
+		sudo cp msvcr120.dll $HOME/.wine32/drive_c/windows/system32/
+		$WINE pip install --upgrade pip
+
+	fi
+}
+
+function install_pyqt(){
+	
+	echo "Download PyQT"
+	if [ ${MACHINE_TYPE} == 'x86_64' ]; then
+		# For 64 bit machine
+		rm -rf PyQt4-4.11.4-gpl-Py2.7-Qt4.8.7-x64.exe
+		rm -rf Cross_Compilation
+  		git clone  https://github.com/surbhicis/Cross_Compilation
+  		$WINE Cross_Compilation/PyQt4-4.11.4-gpl-Py2.7-Qt4.8.7-x64.exe /silent /verysiling /sp- /suppressmsgboxes
```

Comment:
this still seems to behave interactively

Reply:
`/verysiling` is probably a typo